From 30267f5de6f71007d0daac294eac4930a0854671 Mon Sep 17 00:00:00 2001
From: Thomas Richard <thomas.richard@bootlin.com>
Date: Fri, 20 Dec 2024 09:52:16 +0100
Subject: [PATCH] net: stmmac: fix compile issue

Remove the use of wol_irq_disabled in stmmac_request_irq_single() which
does not exist anymore.

Signed-off-by: Thomas Richard <thomas.richard@bootlin.com>
---
 drivers/net/ethernet/stmicro/stmmac/stmmac_main.c | 1 -
 1 file changed, 1 deletion(-)

--- a/drivers/net/ethernet/stmicro/stmmac/stmmac_main.c
+++ b/drivers/net/ethernet/stmicro/stmmac/stmmac_main.c
@@ -3697,7 +3697,6 @@ static int stmmac_request_irq_single(str
 	/* Request the Wake IRQ in case of another line
 	 * is used for WoL
 	 */
-	priv->wol_irq_disabled = true;
 	if (priv->wol_irq > 0 && priv->wol_irq != dev->irq) {
 		ret = request_irq(priv->wol_irq, stmmac_interrupt,
 				  IRQF_SHARED, dev->name, dev);
